### DISCUSSION TITLE
为什么 Alist WebDAV 无法同步文件的修改时间 mtime？

### DISCUSSION THREAD

**Discussion by Coding-Yuan** [remotely-save/remotely-save#643] [Q&A]
在 Alist 上挂载一个本地目录开启了 WebDAV 服务，使用 Remotely Save 同步到这个 Alist WebDAV 时，文件的修改时间总是为同步的时间，怎样才能同步文件原本修改时间？

已试过 Rclone 同步到 Alist 是可以同步文件修改时间的，因为 `vendor = other` 改成了 `vendor = fastmail`。

但是 Remotely Save 直传 Alist WebDAV 却不能保留文件的修改时间。

**Comment by fyears** [comment]
理由很简单，因为这个不是标准设置，目前没有实现。可能会在之后的版本考虑实现

**Comment by fyears** [comment]
可以提交一个 issue 哈
